sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1023,10 +973,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
       );
     }
 
-    if (
-      appToOverride.applicationStatus !== ApplicationStatus.enrollment ||
-      appToOverride.coeStatus !== COEStatus.required
```

Comment:
I am trying to understand, why `appToOverride.coeStatus !== COEStatus.required` check was removed?

Reply:
There is no coe_status in the application anymore. So it has been removed. 
So for the override we are currently checking application to be in enrollment status. 

But there is small issue in just checking application enrollment status.
Here is a scenario when an application is in Enrollment status and has one COE, and the COE is denied.
Now technically we do not update the status of application.
Now if we see on a API perspective this application is eligible for COE override because, the application status is still Enrollment and it gets totally qualified for the override.

We will discuss this one as a team to see what can be done.

Reply:
We added this logic in disbursement schedule service through query as discussed in sync up.